pandas/core/ops/__init__.py

```diff
@@ -25,7 +25,10 @@
     get_array_op,
     logical_op,
 )
-from pandas.core.ops.common import unpack_zerodim_and_defer  # noqa:F401
+from pandas.core.ops.common import (  # noqa:F401
```

Comment:
sure i assume that you want to remove the noqa entirely from here

Reply:
yah